src/storage/admin/AdminTaskManager.cpp

```diff
@@ -353,5 +372,40 @@ bool AdminTaskManager::isFinished(JobID jobID, TaskID taskID) {
   return iter->second->unFinishedSubTask_ == 0;
 }
 
+void AdminTaskManager::cancelSpaceJobs(GraphSpaceID spaceId) {
+  auto it = tasks_.begin();
```

Comment:
will the iterator invalidated by other thread who use `tasks_` at the same time?